src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
```

Comment:
Ensure that the variable `$gap` is defined in your SCSS files. If it's missing, it will cause a compilation error.